sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/job_service.py

```diff
@@ -37,7 +38,9 @@ def __init__(
         self,
         *,
         endpoint: Optional[str] = None,
-        job_service_type: Optional[Literal["JupyterLab", "SSH", "TensorBoard", "VSCode"]] = None,
+        # TODO: Is VS Code allowed here?
+        # TODO: As this is a breaking change, should we still allow the old format (any-case) and deprecate in the next release?
+        job_service_type: Optional[Literal["jupyter_lab", "ssh", "tensor_board", "vs-code"]] = None,
```

Comment:
any reason this is vs-code and not vs_code?

Reply:
This feature was in private preview only. So breaking change should be okay I guess at this point if we want to stick to consistency?